pkg/appmgmt/sparkoperator/spark.go

```diff
@@ -104,6 +105,12 @@ func (os *Manager) Stop() {
 	os.stopCh <- struct{}{}
 }
 
+func (os *Manager) HandleCallbackEvents() func(obj interface{}) {
+	return func(obj interface{}) {
+		//noop
+	}
+}
+
```

Comment:
this can be removed once the handler function is moved to `AppManagementService`.